taskcluster/ci/bump/kind.yml

```diff
@@ -0,0 +1,51 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: taskgraph.loader.transform:loader
+
+kind-dependencies: []
+
+transforms:
+    - rb_taskgraph.transforms.secrets:transforms
+    - taskgraph.transforms.job:transforms
+    - taskgraph.transforms.task:transforms
+
+job-defaults:
+    description: Build Reference Browser from source code.
+    treeherder:
+        kind: build
+        symbol: bump(a-c)
+        tier: 2
+        platform: android-all/opt
+    run-on-tasks-for: []
+    run:
+        dummy-secrets:
+            by-level:
+                '3': []
+                default:
+                    - content: "faketoken"
+                      path: .github_token
+        secrets:
+            by-level:
+                '3':
+                    - name: project/mobile/github
+                      key: botAccountToken
+                      path: .github_token
+                default: []
```

Comment:
Out of scope for this PR, but I'd consider combining `secrets` and `dummy-secrets`, and having the stanza either have name+key or content.

Or, alternatively, having the tasks notice the lack of token, rather than a token with sentinel value.

Reply:
Sounds good! I didn't like this `secrets`/`dummy-secrets` split. I like your solution! I'll give it a try 👍